docassemble/ALToolbox/data/static/review_widget.js

```diff
@@ -0,0 +1,56 @@
+var al_fade_speed = 300;
+
+function after_click() {
+  var thumbs_stage = document.getElementsByClassName('al-thumbs-widget');
+  var feedback_stage = document.getElementsByClassName('al-review-text');
+
+  for (let elem of thumbs_stage) {
+    $(elem).fadeOut(al_fade_speed);
+  }
+  setTimeout(function() {
+    for (let new_elem of feedback_stage) {
+      $(new_elem).fadeIn(al_fade_speed);
+    }
+  }, al_fade_speed);
+}
+
+function after_send() {
+  var feedback_stage = document.getElementsByClassName('al-review-text');
+  var all_done_stage = document.getElementsByClassName('al-after-review');
+
+  for (let elem of feedback_stage) {
+    $(elem).fadeOut(al_fade_speed);
+  }
+  setTimeout(function() {
+    for (let new_elem of all_done_stage) {
+      $(new_elem).fadeIn(al_fade_speed);
+    }
+  }, al_fade_speed);
+}
+
+function altoolbox_thumbs_up_send(event_name, has_feedback) {
+  action_call(event_name);
+  after_click();
+  if (!has_feedback) {
```

Comment:
The argument handed in for this is either 'true' or 'false'. `!has_feedback` checks for a truthy value. If the value was 'false' - a string, which is a truthy value, which would create the opposite behavior than what is desired.

Reply:
It's confusing, but the values in _python_ are either `'true'` or `'false'`.  Copying the line here:

```python
 js_thumbs_up = f"javascript:altoolbox_thumbs_up_send('{up_action}', {'true' if review_action else 'false'})"
 ```
 
 `'true'` and `'false'` are within the curly braces (`{`), which evaluate as python expressions, so it gets added in the surrounding as the exact string and not the python capitalized `True` and `False` (which is why I think it's like that in the first place).